internal/providers/terraform/aws/config_config_rule.go

```diff
@@ -0,0 +1,257 @@
+				VendorName:    strPtr("aws"),
+				Region:        strPtr(region),
+				Service:       strPtr("AWSConfig"),
+				ProductFamily: strPtr("Management Tools - AWS Config Rules"),
+				AttributeFilters: []*schema.AttributeFilter{
+					{Key: "usagetype", ValueRegex: strPtr("/ConfigRuleEvaluations/")},
+				},
+			},
+			PriceFilter: &schema.PriceFilter{
+				StartUsageAmount: strPtr("500000"),
+			},
+		})
+	}
+
+	if packsTierOne.GreaterThan(decimal.NewFromInt(0)) {
+		costComponents = append(costComponents, &schema.CostComponent{
+			Name:            "Conformance pack evaluations (first 1M)",
+			Unit:            "evaluations",
+			UnitMultiplier:  1,
+			MonthlyQuantity: &packsTierOne,
+			ProductFilter: &schema.ProductFilter{
+				VendorName:    strPtr("aws"),
+				Region:        strPtr(region),
+				Service:       strPtr("AWSConfig"),
+				ProductFamily: strPtr("Management Tools - AWS Config Conformance Packs"),
+				AttributeFilters: []*schema.AttributeFilter{
+					{Key: "usagetype", ValueRegex: strPtr("/ConformancePackEvaluations/")},
+				},
+			},
+			PriceFilter: &schema.PriceFilter{
+				StartUsageAmount: strPtr("0"),
+				EndUsageAmount:   strPtr("1000000"),
+			},
+		})
+	}
+
+	if packsTierTwo.GreaterThan(decimal.NewFromInt(0)) {
+		costComponents = append(costComponents, &schema.CostComponent{
+			Name:            "Conformance pack evaluations (next 24M)",
+			Unit:            "evaluations",
+			UnitMultiplier:  1,
+			MonthlyQuantity: &packsTierTwo,
+			ProductFilter: &schema.ProductFilter{
+				VendorName:    strPtr("aws"),
+				Region:        strPtr(region),
+				Service:       strPtr("AWSConfig"),
+				ProductFamily: strPtr("Management Tools - AWS Config Conformance Packs"),
+				AttributeFilters: []*schema.AttributeFilter{
+					{Key: "usagetype", ValueRegex: strPtr("/ConformancePackEvaluations/")},
+				},
+			},
+			PriceFilter: &schema.PriceFilter{
+				StartUsageAmount: strPtr("1000000"),
+				EndUsageAmount:   strPtr("25000000"),
+			},
+		})
+	}
+
+	if packsTierThree.GreaterThan(decimal.NewFromInt(0)) {
+		costComponents = append(costComponents, &schema.CostComponent{
+			Name:            "Conformance pack evaluations (over 25M)",
+			Unit:            "evaluations",
+			UnitMultiplier:  1,
+			MonthlyQuantity: &packsTierThree,
+			ProductFilter: &schema.ProductFilter{
+				VendorName:    strPtr("aws"),
+				Region:        strPtr(region),
+				Service:       strPtr("AWSConfig"),
+				ProductFamily: strPtr("Management Tools - AWS Config Conformance Packs"),
+				AttributeFilters: []*schema.AttributeFilter{
+					{Key: "usagetype", ValueRegex: strPtr("/ConformancePackEvaluations/")},
+				},
+			},
+			PriceFilter: &schema.PriceFilter{
+				StartUsageAmount: strPtr("25000000"),
+			},
+		})
+	}
+
+	return &schema.Resource{
+		Name:           d.Address,
+		CostComponents: costComponents,
+	}
+}
+
+func calculateTierItems(items decimal.Decimal, tiers map[string]decimal.Decimal, tierLimits []int64) map[string]decimal.Decimal {
```

Comment:
Just checking if you have seen https://github.com/infracost/infracost/issues/389 ? I wonder if we could do that ticket and share that logic? Sorry I haven't looked at this PR in details yet